etc/ex.mock.config.yaml

```diff
@@ -17,3 +17,6 @@ openshift:
   target: {{OPENSHIFT_TARGET}}
   user: {{OPENSHIFT_USER}}
   pass: {{OPENSHIFT_PASS}}
+broker:
```

Comment:
Fine with adding the config sections in now if you want.  

Going forward the runtime config will be generated from the Template in a ConfigMap, so these config files only serve as reference points.

If we want multiple config examples for various situations or a single with comments, fine with either.